internal/services/containers/container_registry_token_password_resource.go

```diff
@@ -453,14 +454,23 @@ PasswordGenLoop:
 			return nil, fmt.Errorf("generating password credential %s: %v", string(*password.Name), err)
 		}
 
-		res, err := clients.ContainerRegistryClient_v2021_08_01_preview.Registries.ListCredentials(ctx, registryId)
+		result, err := clients.ContainerRegistryClient_v2021_08_01_preview.Registries.GenerateCredentials(ctx, registryId, param)
```

Comment:
this would cycle the credentials however?

Reply:
Wow... Forgot to delete the line above.. Thank you for catching this!